<?xml version="1.0"?>
<launch>

	<arg name="acceleration_limit" default="3.0"/>
	<arg name="deceleration_limit" default="3.0"/>
	<arg name="max_curvature_rate" default="0.75"/>

	<node pkg="aev" type="aev_interface" name="aev_interface" output="screen">
		<param name="acceleration_limit" value="$(arg acceleration_limit)"/>
		<param name="deceleration_limit" value="$(arg deceleration_limit)"/>
		<param name="max_curvature_rate" value="$(arg max_curvature_rate)"/>
	</node>

</launch>
